### PR TITLE
Make httputil.Client support cookies.

### PR DESCRIPTION
**Pull request by gjoliver**

### CODE REVIEW

**Review by h8liu — APPROVED** [review]
do you also need some way to read cookie?

**Comment by h8liu** [comment]
> do you also need some way to read cookie?

I mean read cookie from a response?

File: httputil/client.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ type Client struct {
 	Accept    string // Optional Accept header.
 
 	Transport http.RoundTripper
+	Cookies   []*http.Cookie
```
Comment (h8liu) [inline httputil/client.go:26]: in fact, you can probably use this:

https://golang.org/pkg/net/http/cookiejar/

**Comment by gjoliver** [comment]
don't have the need for response cookie yet.
that feels like a bigger change, since we don't pass the raw response out?

**Pull request closed without merging**